src/napari_spatialdata/utils/_utils.py

```diff
@@ -178,25 +178,16 @@ def _swap_coordinates(data: list[Any]) -> list[Any]:
     return [[(y, x) for x, y in sublist] for sublist in data]
 
 
-def _get_transform(element: SpatialElement, coordinate_system_name: str | None = None) -> NDArrayA:
-    affine: NDArrayA
-    transformations = get_transformation(element, get_all=True)
-    cs = transformations.keys().__iter__().__next__() if coordinate_system_name is None else coordinate_system_name
-    ct = transformations[cs]
-    affine = ct.to_affine_matrix(input_axes=("y", "x"), output_axes=("y", "x"))
-
+def _get_transform(element: SpatialElement, coordinate_system_name: str | None = None) -> None | NDArrayA:
     if not isinstance(element, (SpatialImage, MultiscaleSpatialImage, AnnData, DaskDataFrame, GeoDataFrame)):
```

Comment:
I removed AnnData, it could have transformations associated to it only in a very old version of the SpatialData class

Reply:
Ah you removed before me :P